docs/docs/Contributing/contributing-telemetry.md

```diff
@@ -4,33 +4,15 @@ sidebar_position: 0
 slug: /contributing-telemetry
 ---
 
-
-
-:::info
-
-This page may contain outdated information. It will be updated as soon as possible.
-
-:::
-
-
-
-
-Our system uses anonymous telemetry to collect essential usage statistics to enhance functionality and user experience. This data helps us identify commonly used features and areas needing improvement, ensuring our development efforts align with what you need.
-
-
-:::note
+Langflow uses anonymous telemetry to collect essential usage statistics to enhance functionality and user experience. This data helps us identify popular features and areas needing improvement, and ensures development efforts align with what you need.
 
 We respect your privacy and are committed to protecting your data. We do not collect any personal information or sensitive data. All telemetry data is anonymized and used solely for improving Langflow.
 
 You can opt-out of telemetry by setting the `LANGFLOW_DO_NOT_TRACK` or `DO_NOT_TRACK` environment variable to `true` before running Langflow. This will disable telemetry data collection.
```

Comment:
I think this information gets lost. It should be under a heading "Opt out of telemetry". Also, remove the hypen from "opt out" in the sentence.